geomesa-index-api/src/test/scala/org/locationtech/geomesa/index/index/AttributeIndexTest.scala

```diff
@@ -354,5 +357,23 @@ class AttributeIndexTest extends Specification with LazyLogging {
             features.slice(2, 3)
       }
     }
+
+    "handle filter.EXCLUDE with query hint" in {
```

Comment:
Huh, this wasn't working before?

Reply:
no, i guess people don't query on Filter.EXCLUDE a lot :P

Reply:
Hehehhee.